background-concealment-demo-py/BackgroundConcealmentDemo.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    g_nn_outs = cv.gapi.infer(nn_id, g_nn_ins)
</pre><p> </p><p>As in one of previous comments - maybe explicit arguments names could be useful for a user here</p><pre class="ql-syntax" spellcheck="false">g_nn_outs = cv.gapi.infer(net_id=nn_id, net_ins=cv.GInferInputs().setInput('input', g_in))
</pre><p> </p><p>And in some other kernel calls in this part as well</p><pre class="ql-syntax" spellcheck="false">g_alpha&nbsp; &nbsp; = cv.gapi.resize(in=GTensorToImg.on(g_nn_out), size=in_size)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TolyaTalamanov/g-api-collection/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>